src/Symfony/Component/Config/Loader/FileLoader.php

```diff
@@ -79,6 +79,8 @@ public function getLocator()
      *
      * @throws FileLoaderLoadException
      * @throws FileLoaderImportCircularReferenceException
+     * @throws FileLocatorFileNotFoundException
+     * @throws \LogicException
```

Comment:
I removed the `\LogicException`
The other should be moved to the other  method (The internal `glob` method) ? which throws all these exceptions